examples/pytorch/cluster_gcn/cluster_gcn.py

```diff
@@ -138,8 +136,8 @@ def main(args):
     # set train_nids to cuda tensor
     if cuda:
         train_nid = torch.from_numpy(train_nid).cuda()
-    print("current memory after model before training",
-          torch.cuda.memory_allocated(device=train_nid.device) / 1024 / 1024)
+        print("current memory after model before training",
```

Comment:
control `print` with `self.verbose`

Reply:
This code is in example, and it's a bug fix. When running with CPU, this code failed.